src/react/account/AccountContent.tsx

```diff
@@ -65,13 +61,11 @@ function AccountContent() {
             <AccountHead />
```

Comment:
Instead of using the legacy `<AccountHead>` we could try something as below:
```js
<Stack gap="r16">
  <Icon name="Account" color="infoPrimary" size="2x" withWrapper />
  <Text variant="Larger">{accountNameParam}</Text>
</Stack>
```

In general, what I'd like is to get rid of the ListLayout / ListLayout2 / ListLayout3 / ListLayout4 / ListLayout5 as much as possible since we have the new Layout component, could you check that please?